src/test/groovy/com/palantir/gradle/gitversion/GitVersionPluginTests.groovy

```diff
@@ -129,12 +132,13 @@ class GitVersionPluginTests extends Specification {
         git.add().addFilepattern('.').call()
         git.commit().setMessage('initial commit').call()
         dirtyContentFile << 'dirty-content'
+        String expected = git.getRepository().getRef("HEAD").getObjectId().abbreviate(VERSION_ABBR_LENGTH).name()
```

Comment:
can you turn this into a method, and also keep the VERSION_ABBR_LENGTH variable local to that method?